service/lib/agama/storage/finisher.rb

```diff
@@ -273,6 +273,7 @@ def collect_logs
           Yast::Execute.locally(
             "agama", "logs", "store", "--destination", path
           )
+          FileUtils.chmod(0o640, Dir.glob("#{path}*"))
```

Comment:
in yast usually private files use umask 077 so here it would be 0o600, so really only root can read it.